### PR TITLE
UI: add tables as unique rows

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
Removes the business of checking if we've expanded a schema and then inserting rows or not
Simpler: uses lodash functions to attain a unique array of objects while inserting table rows on schema expand

### CODE REVIEW

**Review by iskakaushik — APPROVED** [review]

**Comment by serprex** [comment]
Use `lodash-es` instead so that tree shaking can reduce bundle size impact

File: ui/app/mirrors/create/cdc/schemabox.tsx [hunk 1]
```diff
@@ -136,14 +134,11 @@ const SchemaBox = ({
...
-        setTablesLoading(true);
-        setSchemaLoadedSet((loaded) => new Set(loaded).add(schemaName));
-        fetchTables(sourcePeer, schemaName, peerType).then((tableRows) => {
-          setRows((value) => [...value, ...tableRows]);
-          setTablesLoading(false);
-        });
-      }
+      setTablesLoading(true);
+      fetchTables(sourcePeer, schemaName, peerType).then((tableRows) => {
+        setRows((value) => _.uniqWith([...value, ...tableRows], _.isEqual));
```
Comment (serprex) [inline ui/app/mirrors/create/cdc/schemabox.tsx:139]: Should be straight forward to roll a utility function to achieve this same effect without bringing in lodash dependency. Uniqueness could be keyed on a generated string (maybe join fields with `\n`) avoiding `isEqual` cost. At worst sort & dedupe

Comment (serprex) [reply ui/app/mirrors/create/cdc/schemabox.tsx:139]: https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L8441
https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L4249

By passing `_.isEqual` lodash is forced to use `O(N*N)` algo

**Review by serprex — CHANGES_REQUESTED** [review]
more efficient algo should be used

**Review by serprex — APPROVED** [review]

**Pull request merged**